omrsigcompat/makefile

```diff
@@ -25,6 +25,20 @@ OBJECTS := omrsig
 OBJECTS := $(addsuffix $(OBJEXT),$(OBJECTS))
 EXPORT_FUNCTIONS_FILE := omrsig.exportlist
 
+ifeq (win,$(OMR_HOST_OS))
+COPYRIGHT_RC_FILE := ../../omrsigcompat/omrsig.rc
+COPYRIGHT_RES_FILE := omrsig.res
```

Comment:
where do these files come from? Is this sort of DLL copyright information added to other OMR DLLs?